modules/@apostrophecms/asset/index.js

```diff
@@ -46,12 +46,12 @@ module.exports = {
     // disable the build of the public UI assets.
     publicBundle: true,
     // Device preview in the admin UI.
-    // NOTE: the whole devicePreviewMode option must be carried over
-    // to the project for override to work properly.
+    // NOTE: the whole breakpointPreviewMode option must be carried over
+    // to the project for overrides to work properly.
     // Nested object options are not deep merged in Apostrophe.
-    devicePreviewMode: {
-      // Enable device preview mode
-      enable: false,
+    breakpointPreviewMode: {
+      // Enable breakpoint preview mode
+      enable: true,
```

Comment:
Please tell me if we're missing something, e.g. we think this feature is highly likely to break somebody's build completely if we default it to true. Otherwise we really want people to have this experience out of the box

Reply:
We do some css transformations on css declarations inside media queries.
If you have 

```css
@media (max-width: 700px) {
  main {
    width: 100%;
  }
}
```

you will end up with

```css
@media (max-width: 700px) {
  :where(:not(body[data-breakpoint-preview-mode])) main {
    width: 100%;
  }
}
@container (max-width: 700px) {
  main {
    width: 100%;
  }
}
```

`:where` is widely supported https://developer.mozilla.org/en-US/docs/Web/CSS/:where

I should be safe to use.

Reply:
that was the main driver for setting `enabled: false` by default.